webservices/resources/committees.py

```diff
@@ -0,0 +1,168 @@
+    'link_date': fields.String,
+    'expire_date': fields.String,
+}
+committee_fields = {
+    'committee_id': fields.String,
+    'name': fields.String,
+    'designation_full': fields.String,
+    'designation': fields.String,
+    'treasurer_name': fields.String,
+    'organization_type_full': fields.String,
+    'organization_type': fields.String,
+    'state': fields.String,
+    'party_full': fields.String,
+    'party': fields.String,
+    'committee_type_full': fields.String,
+    'committee_type': fields.String,
+    'expire_date': fields.String,
+    'original_registration_date': fields.String,
+    'candidates': fields.Nested(candidate_commitee_fields),
+}
+pagination_fields = {
+    'per_page': fields.Integer,
+    'page': fields.Integer,
+    'count': fields.Integer,
+    'pages': fields.Integer,
+}
+committee_list_fields = {
+    'api_version': fields.Fixed(1),
+    'pagination': fields.Nested(pagination_fields),
+    'results': fields.Nested(committee_fields),
+}
+
+
+class CommitteeList(Resource):
+    parser = reqparse.RequestParser()
+    parser.add_argument('q', type=str, help='Text to search all fields for')
+    parser.add_argument('committee_id', type=str, help="Committee's FEC ID")
+    parser.add_argument('state', type=str, help='Two digit U.S. State committee is registered in')
+    parser.add_argument('name', type=str, help="Committee's name (full or partial)")
+    parser.add_argument('page', type=int, default=1, help='For paginating through results, starting at page 1')
+    parser.add_argument('per_page', type=int, default=20, help='The number of results returned per page. Defaults to 20.')
+    parser.add_argument('committee_type', type=str, help='The one-letter type code of the organization')
+    parser.add_argument('designation', type=str, help='The one-letter designation code of the organization')
+    parser.add_argument('organization_type', type=str, help='The one-letter code for the kind for organization')
+    parser.add_argument('party', type=str, help='Three letter code for party')
+    parser.add_argument('expire_date', type=str, help='Date the committee registration expires')
+    parser.add_argument('original_registration_date', type=str, help='Date of the committees first registered')
+    parser.add_argument('candidate_id', type=str, help='FEC IDs of candidates that committees have mentioned in filings. (ee designation for the nature of the relationship.)')
+    parser.add_argument('year', type=str, default=None, help='A year that the committee was active- (fter original registration but before expiration.)')
+
+    @marshal_with(committee_list_fields)
+    def get(self, **kwargs):
+
+        args = self.parser.parse_args(strict=True)
+
+        # pagination
+        page_num = args.get('page', 1)
+        per_page = args.get('per_page', 20)
+
+        count, committees = self.get_committees(args, page_num, per_page)
+
+        data = {
+            'api_version': '0.2',
```

Comment:
Might be a good idea to move this into a shared function at some point -- also, is it actually changing?